test/smoke/testApps/FixedRateSampling/src/smokeTest/resources/appServers.txt

```diff
@@ -0,0 +1,6 @@
+jbosseap6
```

Comment:
I believe @littleaj also added support for WebSphere. If so please do add that too.

Reply:
@dhaval24 I didn't see WebSphere in appServers folder. @littleaj Did you add it?

Reply:
Not yet